<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSetting</class>
 <widget class="QMainWindow" name="DeviceSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceSetting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="4">
       <widget class="QPushButton" name="m_pushbutton_SettingDevice">
        <property name="text">
         <string>配置设备</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QChartView" name="m_chartview"/>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="m_pushbutton_OpenCom">
        <property name="text">
         <string>打开串口</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="m_comboBox_BautSelect"/>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="m_radioButton_filter">
        <property name="text">
         <string>滤波</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="m_pushbutton_OpenDevice">
        <property name="text">
         <string>开启设备</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="m_pushbutton_CloseDevice">
        <property name="text">
         <string>关闭设备</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="m_comboBox_ChannelSelect"/>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="m_comboBox_freq"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="m_comboBox_ComNameSelect"/>
      </item>
      <item row="4" column="3">
       <widget class="QRadioButton" name="m_radioButton_record">
        <property name="text">
         <string>数据记录</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="m_radioButton_raw_record">
        <property name="text">
         <string>原始数据</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLCDNumber" name="m_lcdNumber_frame_count">
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="draw_switch"/>
    <addaction name="cmd_switch"/>
    <addaction name="separator"/>
    <addaction name="chart_setting"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="draw_switch">
   <property name="text">
    <string>绘图：开</string>
   </property>
  </action>
  <action name="cmd_switch">
   <property name="text">
    <string>控制台：开</string>
   </property>
   <property name="toolTip">
    <string>控制台：开</string>
   </property>
  </action>
  <action name="chart_setting">
   <property name="text">
    <string>绘图设置</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
